ext/OceananigansXESMFExt.jl

```diff
@@ -9,13 +9,21 @@ using Oceananigans.Architectures: on_architecture, CPU
 
 import Oceananigans.Fields: regridding_weights
 
-x_node_array(x::AbstractVector, Nx, Ny) = view(x, 1:Nx) |> Array
-y_node_array(x::AbstractVector, Nx, Ny) = view(x, 1:Ny) |> Array
-x_node_array(x::AbstractMatrix, Nx, Ny) = view(x, 1:Nx, 1:Ny) |> Array
+function x_node_array(x::AbstractVector, Nx, Ny)
```

Comment:
```suggestion
function x_node_matrix(x::AbstractVector, Nx, Ny)
```
I guess this is really a matrix rather than an array then

Reply:
hm...  perhaps I missed your point
yes, all the coords need to be 2D arrays before we give them to xesmf.Regridder
but what do you mean "matrix" instead of "array"?
I guess matrix is more precise since Array can be 1D, 2D, 3D, ... Is this what you were referring to?